_data/merchants.yml

```diff
@@ -209,6 +211,10 @@
       url: http://investmentart.org/
     - name: Mushroom cultures, mushroom growing supplies, seeds
       url: https://www.vesp.co/Fungible
+    - name: Plazmatic
+      url: http://www.plazmatic.com
+    - name: Project Coral Reef Store
+      url: https://shop.projectcoralreef.com
```

Comment:
Would be better to change this to `http://www.projectcoralreef.com/all-stores/`, see following comment.